csharp/Microsoft.Azure.Databricks.Client/Models/WorkspaceStorageInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Azure.Databricks.Client.Models;
+
+public record WorkspaceStorageInfo
+{
+    /// <summary>
+    /// Workspace destination, e.g. /my/path
+    /// </summary>
+    [JsonPropertyName("destination")]
+    public string Destination { get; set; }
+}
```

Comment:
There is already a DbfsStorageInfo with the same structure. Maybe rename that one to "InitScriptStorageInfo" and reuse it for dbfs, workspace and volumns? They all have the same structure.